src/registrar/admin.py

```diff
@@ -4,13 +4,28 @@
 from django.contrib.contenttypes.models import ContentType
 from django.http.response import HttpResponseRedirect
 from django.urls import reverse
+from registrar.models.utility.admin_form_order_helper import AdminFormOrderHelper
```

Comment:
I had it that way originally but the linter was not happy about the length, would it make sense to add a # noqa tag here?

Reply:
Linter complained about the length, that is no fun. I lean to the #noqa tag vs separating it into two imports. However, because it is to get around  lint if you want to keep it the same way you had it that is fine, I would just update your comment so it is clearit's sepateing due to the line length if combined. Otherwise, I could totally see an engineer coming around later and trying to consolidate them.